config/environments/production.rb

```diff
@@ -86,4 +86,9 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.web_socket_server_url = ENV.fetch('WEB_SOCKET_SERVER_URL',
+                                           'wss://jm-target-mvd.herokuapp.com/cable')
+  config.action_cable.allowed_request_origins = ENV.fetch('ACTION_CABLE_ALLOWED_REQUEST_ORIGINS',
```

Comment:
Hmmm why this? on production localhost:3000 🤔

Reply:
You're right, doesn't make any sense! 🙈